tests/src/test/scala/common/Wsk.scala

```diff
@@ -71,6 +71,8 @@ import whisk.utils.retry
 
 case class WskProps(
     authKey: String = WhiskProperties.readAuthKey(WhiskProperties.getAuthFileForTesting),
+    cert: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-cert.pem",
+    key: String = WhiskProperties.getProperty("openwhisk.home") + "/ansible/roles/nginx/files/openwhisk-client-key.pem",
```

Comment:
Can you show the method, if doesn't exist, how about add a new method  in `WhiskProperties` like below
```
 /**
     * The path to the whisk home directory.
     */
    public static String getCLIDir() {
        return whiskHome;
    }